lib/src/components/NxRadio/NxRadio.tsx

```diff
@@ -30,8 +30,8 @@ const NxRadio = forwardRef<HTMLLabelElement, Props>(
                  onChange={() => onChange && onChange(value)}
                  readOnly={!onChange}/>
           <svg className="nx-radio__circle" viewBox="-8 -8 16 16" focusable={false}>
-            <circle r="7" strokeWidth="1" fill="none" className="nx-radio__outer-circle"/>
-            { isChecked && <circle r="5" stroke="none" className="nx-radio__inner-circle"/> }
+            <circle r="7" strokeWidth="1" className="nx-radio__outer-circle"/>
```

Comment:
SVG stroke width emanates equally from both sides of the stroke.  Therefore, to get a circle with a stroke width of 1 and an _outer_ radius of 8 (which I believe is what zeplin is specifying), set the radius to 7.5.
```suggestion
            <circle r="7.5" strokeWidth="1" className="nx-radio__outer-circle"/>
```